sympy/algebras/quaternion.py

```diff
@@ -720,3 +720,297 @@ def to_rotation_matrix(self, v=None):
 
             return Matrix([[m00, m01, m02, m03], [m10, m11, m12, m13],
                           [m20, m21, m22, m23], [m30, m31, m32, m33]])
+
+    def scalar(self):
+        """
+        Returns the scalar part of the quaternion.
+
+        Returns
+        =======
+
+        The scalar part of the quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(1, 1, 1, 1)
+        >>> q.scalar()
+        1
+
+        >>> q = Quaternion(4, 8, 13, 12)
+        >>> q.scalar()
+        4
+
+        """
+
+        return self.a
+
+    def vector(self):
+        """
+        Returns the vector part of the quaternion.
+
+        Returns
+        =======
+
+        V : The vector part of the quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(1, 1, 1, 1)
+        >>> q.vector()
+        0 + 1*i + 1*j + 1*k
+
+        >>> q = Quaternion(4, 8, 13, 12)
+        >>> q.vector()
+        0 + 8*i + 13*j + 12*k
+
+        """
+
+        return Quaternion(0, self.b, self.c, self.d)
+
+    def axis(self):
+        """
+        Returns the axis that is the versor of the vector part of the quaternion.
+
+        Returns
+        =======
+
+        The axis of the quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(1, 1, 1, 1)
+        >>> q.axis()
+        0 + sqrt(3)/3*i + sqrt(3)/3*j + sqrt(3)/3*k
+
+        >>> q = Quaternion(4, 8, 13, 12)
+        >>> q.axis()
+        0 + 8*sqrt(377)/377*i + sqrt(377)/29*j + 12*sqrt(377)/377*k
+
+        """
+
+        return Quaternion(0, self.b, self.c, self.d).normalize()
+
+    def is_pure(self):
+        """
+        Returns true if the quaternion is pure else returns false.
```

Comment:
This is fine as a one sentence summary but there should be a longer explanation of what the definition of "pure" is in an "explanation" section.